pkg/schema/schema.json

```diff
@@ -75,6 +80,12 @@
                             }
                         ]
                     }
+                },
+                "topLevelDomains": {
+                    "type": "array",
+                    "title": "Top-Level-Domains",
+                    "description": "A list of TLDs which may be present in the input must-gather",
+                    "default": [".com", ".org", ".info", ".gov"]
```

Comment:
is there a big TLD list that we can just checkin once?

Reply:
There probably is, but there are many TLDs which are trademarked names and I didn't want to add them. Users can add whatever TLDs they use themselves.

Reply:
well, we don't need to include `.fiat`, but can you please add the country codes and the remainders of the original ICANN set? https://en.wikipedia.org/wiki/Country_code_top-level_domain